src/main/resources/db/migration/V2__Add_informtion.sql

```diff
@@ -0,0 +1,26 @@
+insert into DRIVER
+values (1, 'Gwion Ritter', '1234567890', '142359708', '89112345678', 'login', 'password'),
```

Comment:
After you add serial or sequence for ID column, we don't need to set ID explicitly, so remove them